tests/conftest.py

```diff
@@ -41,8 +41,8 @@ def pytest_collection_modifyitems(config, items):
             item.add_marker(skip_slow)
 
 
-@pytest.fixture(scope="session", autouse=True)
-def mock_versioning_api():
+@pytest.fixture(scope="module", autouse=True)
```

Comment:
Applies fixture per test module. Cannot use per session as we have to overwrite fixture for the version check tests.